fix SUID/SGID bit cleaning API spelling (unkown -> unknown)

dupuy opened the pull request:
Having a misspelling in an API is especially awkward, since fixing the spelling could break the code. But I think this is obscure enough that nobody is likely to be using it already...

chris-rock commented:
looks great. I think we need to fix the implementation, too: https://github.com/TelekomLabs/chef-os-hardening/blob/master/recipes/suid_sgid.rb#L43

chris-rock commented:
thanks @dupuy for pointing this out!

chris-rock commented:
@dupuy to make it complete, it would be great if you could fix it in https://github.com/TelekomLabs/chef-os-hardening/blob/master/TUTORIAL.md as well

dupuy commented:
OK, I (re?)added the typo fix to TUTORIAL.md, and fixed a rubocop alignment error I introduced when I wrapped a long line - with the additional 2 commits, it is satisfying Travis and Coveralls.

chris-rock commented:
@dupuy great work!
